analysis/druidguardian/src/modules/features/AlwaysBeCasting.tsx

```diff
@@ -0,0 +1,29 @@
+import { t } from '@lingui/macro';
+import { formatPercentage } from 'common/format';
+import { When } from 'parser/core/ParseResults';
+import CoreAlwaysBeCasting from 'parser/shared/modules/AlwaysBeCasting';
+import React from 'react';
+
+class AlwaysBeCasting extends CoreAlwaysBeCasting {
```

Comment:
generally you don't need to override this unless you are adding custom functionality. what differs between this and the base class?

Reply:
Thanks for the review @emallson.  So I added this override here because I wish to expand on the ABC suggestions area for guardian with some tips on how to fill the gaps with MF/IF, and Swipe.  This is a precursor to more changes I am working on.